llama-index-packs/llama-index-packs-corrective-rag/requirements.txt

```diff
@@ -0,0 +1 @@
+tavily-python
```

Comment:
requirements.txt is legacy, in fact i should write a script to remove them all from packs. Now that these are their own python packages and we use poetry to manage our deps, we should use poetry exclusively.

Please do a `poetry add tavily-python` to get it added to your pyproject.toml